### PR TITLE
update doc: make tensorflow.js description more clear

### PR DESCRIPTION
**Pull request by TheEhsanSarshar**

### CODE REVIEW

**Comment by lamberta** [comment]
Can you please double-check that you've signed the CLA? https://cla.developers.google.com/

After signing, add a comment `@googlebot I signed it!` to trigger the bot 🤖

**Comment by TheEhsanSarshar** [comment]
@googlebot I signed it!

File: site/en/install/_index.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ landing_page:
       path: https://js.tensorflow.org
       description: >
         TensorFlow.js is a WebGL accelerated, JavaScript library to train and
-        deploy ML models in the browser and for Node.js.
+        deploy ML models in the browser, Node.js, React native, electron.js, IOT devices and many more.
```
Comment (lamberta) [inline site/en/install/_index.yaml:106]: I think the description on https://www.tensorflow.org/install is getting a bit long.
Can change it to "... in the browser, Node.js, and more."

**Comment by TheEhsanSarshar** [comment]
@googlebot I signed it!

File: site/en/install/_index.yaml (hunk 1 above)
Comment (TheEhsanSarshar) [reply site/en/install/_index.yaml:106]: yeah, it's a little bit longer. 
what if we change it to browser, Node.js, desktop, mobile and many more

Comment (TheEhsanSarshar) [reply site/en/install/_index.yaml:106]: or browser, Node.js, mobile, and more.

Comment (TheEhsanSarshar) [reply site/en/install/_index.yaml]: f85d1a556ae3bade9d67aa1841a51c803e4c64c6

**Review by lamberta — APPROVED** [review]
Thanks!

**Pull request merged**